app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
 
+    people = {}
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
+
+
+def create_person_list(persons: list[dict]) -> list[Person]:
```

Comment:
It’s picky but `persons` Is not really good name of variable. You already have `people` by default